ejercicio 18.cpp

```diff
@@ -0,0 +1,68 @@
+// ejercicio 18.cpp : Este archivo contiene la función "main". La ejecución del programa comienza y termina ahí.
+//
+#include <iostream>
+#include <conio.h>
+using namespace std;
+
+int main()
+{
+    int size = 5;
+
+    // Creación
+    int** matrix = new int* [size];
+
+    int rows = size; // filas
+    int cols = size; // columnas
+
+    int N = size;     // 5
+    int M = size - 2; // 3
+
+    int contador = 1; // 🔹 Numerador

```

Comment:
No corre con este caracter especial, hiciste la prueba ??
```suggestion
    int contador = 1; // Numerador

```